src/BugsnagUnity/ICacheManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using BugsnagUnity.Payload;
+
+namespace BugsnagUnity
+{
+    public interface ICacheManager
```

Comment:
Do you need something here to set the max number of cached events/sessions?

Reply:
the fallback doesnt need that as it has the config but i can see that perhaps the native plugin will need it.
but there are a few things, like setMaxCacheSize that also might be needed so i'm going to only implement what is currently needed in this PR and make changes as needed when the time comes later on 👍